pontoon/base/static/js/lib/fluent-syntax.js

```diff
@@ -1,15 +1,31 @@
-/* fluent-syntax@0.3.0 */
+/* fluent-syntax@0.4.0 */
```

Comment:
We use `FluentSyntax.parse()` a few times in https://github.com/mathjazz/pontoon/blob/HEAD/pontoon/base/static/js/fluent_interface.js.